deployment/src/test/java/io/quarkiverse/openapi/generator/deployment/wrapper/OpenApiClientGeneratorWrapperTest.java

```diff
@@ -387,6 +387,28 @@ void withoutAnyTypeOrImportMappingsItShouldGenerateUsingJava8DatesAndTimes()
                 .contains("java.time.LocalDate", "java.time.OffsetDateTime");
     }
 
+    @Test
+    void shouldBeAbleToEnableMutiny() throws URISyntaxException, FileNotFoundException {
+        List<File> generatedFiles = createGeneratorWrapper("simple-openapi.json")
+                .withMutiny(true)
+                .generate("org.mutiny.enabled");
+
+        Optional<File> file = generatedFiles.stream()
+                .filter(f -> f.getName().endsWith("DefaultApi.java"))
+                .findAny();
+        assertThat(file).isNotEmpty();
+        CompilationUnit compilationUnit = StaticJavaParser.parse(file.orElseThrow());
+        List<MethodDeclaration> methodDeclarations = compilationUnit.findAll(MethodDeclaration.class);
+        assertThat(methodDeclarations).isNotEmpty();
+
+        methodDeclarations.forEach(m -> {
+            var returnType = m.getType().toString();
+            assertTrue(returnType.startsWith("Uni<"));
```

Comment:
Please use the FQCN.

```suggestion
            assertTrue(returnType.startsWith("io.smallrye.mutiny.Uni<"));
```